sirepo/pkcli/hellweg.py

```diff
@@ -12,7 +12,7 @@
 from sirepo.template import template_common
 import copy
 import py.path
-import sirepo.template.rshellweg as template
+import sirepo.template.hellweg as template
```

Comment:
This is not correct style. This should import sirepo.template, and when it needs hellweg, it should:

```py
t = sirepo.template.import_module("hellweg")
```